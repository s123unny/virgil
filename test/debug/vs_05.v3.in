b ../core/vs_05.v3 3
b ../core/vs_05.v3 4
b ../core/vs_05.v3 5
b ../core/vs_05.v3 10
b ../core/vs_05.v3 13
run
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
q
