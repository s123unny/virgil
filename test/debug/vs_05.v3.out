breakpoint|../core/vs_05.v3 3|0
breakpoint|../core/vs_05.v3 4|1
breakpoint|../core/vs_05.v3 5|2
breakpoint|../core/vs_05.v3 10|3
breakpoint|../core/vs_05.v3 13|4
stopOnBreakpoint
bt|main|../core/vs_05.v3|10
variable|0|a|0|int|false
variable|1|o|#1:vs_05<byte>|vs_05<byte>|true
stopOnBreakpoint
bt|vs_05.m|../core/vs_05.v3|3
bt|main|../core/vs_05.v3|10
stopOnBreakpoint
bt|vs_05.n|../core/vs_05.v3|4
bt|main|../core/vs_05.v3|10
stopOnBreakpoint
bt|vs_05.o|../core/vs_05.v3|5
bt|main|../core/vs_05.v3|10
variable|0|this|#1:vs_05<byte>|vs_05<T>|true
end
result|6
